azure/flow-runner-function/src/main/java/com/marklogic/dhf/azure/FunctionDhfFlow.java

```diff
@@ -38,20 +38,25 @@ public HttpResponseMessage run(
         } else {
             // Write the name to the message queue.
             msg.setValue(name);
-
-            runFlow();
-            return request.createResponseBuilder(HttpStatus.OK).body("Hello, " + name).build();
+            String runFlowMessage = "Flow <" + name + "> successfully executed."; //default message
+            try {
+                runFlow(name);
+            } catch (Exception ex) {
+                context.getLogger().info("Exception caught: " + ex.getMessage());
```

Comment:
Good point. User would never see this message as it will be logged at the azure apps backend. But nonetheless, changed to "severe" to honor the semantics.